dialoptions.go

```diff
@@ -235,11 +235,6 @@ func WithServiceConfig(c <-chan ServiceConfig) DialOption {
 // https://github.com/grpc/grpc/blob/master/doc/connection-backoff.md. Consider
```

Comment:
> ...configures the dialer...

What's a "dialer"?  I think this is the wrong term to use.

```go
// WithConnectParams configures the ClientConn to use the provided ConnectParams
// for creating and maintaining connections to servers.
```

Reply:
+1

(And to clarity, this is to replace line 231 with the suggested comment, and github doesn't allow comments on 231 because it's unchanged, and far from where the actual changes are)

Reply:
Thanks for clarifying.  I didn't realize how vague my comment was at the time.